Lombiq.HelpfulLibraries/Libraries/Users/ICachingUserManager.cs

```diff
@@ -10,28 +10,36 @@ namespace Lombiq.HelpfulLibraries.Libraries.Users
     public interface ICachingUserManager
     {
         /// <summary>
-        /// Retrieves <see cref="User"/>s from a transient per-request cache by their unique ID or gets them from the
-        /// store if not yet cached.
+        /// Retrieves <see cref="User"/>s from a transient per-request cache by their <see cref="User.Id"/> or gets them
+        /// from the store if not yet cached.
+        /// </summary>
+        /// <param name="id">Unique ID identifying the <see cref="User"/> document.</param>

```

Comment:
You're missing the `<param name="forceUpdate">` here and in all the methods below.

Reply:
We usually skip the obvious params where nothing is substance could be added.